internal/constants/constants.go

```diff
@@ -161,6 +160,10 @@ const ArtifactCacheFileName = "artifact_cache.json"
 // ArtifactMetaDir is the directory in which we store meta information about artifacts
 const ArtifactMetaDir = "artifacts"
 
+// ArtifactCacheSizeEnvVarName is the maximum size in MB of the artifact cache.
+// The default is 500MB.
+const ArtifactCacheSizeEnvVarName = "ACTIVESTATE_ARTIFACT_CACHE_SIZE_MB"
```

Comment:
Can we move or duplicate this to the `state config` command?

Reply:
As discussed in Slack, this will be in a separate ticket.